<launch>

    <node pkg="usv_control" type="asmc" name="asmc" >
	<param name = "k_u" value = "0.1" />
	<param name = "k_psi" value = "0.2" />
	<param name = "kmin_u" value = "0.05" />
	<param name = "kmin_psi" value = "0.2" />
	<param name = "k2_u" value = "0.02" />
	<param name = "k2_psi" value = "0.1" />
	<param name = "mu_u" value = "0.05" />
	<param name = "mu_psi" value = "0.1" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
    </node>

    <node pkg="usv_control" type="dynamic_model_simulate" name="dynamic_model_simulate" />

    <node pkg="sensors" type="waypoints3.py" name="waypoints3" />

    <node name="los" pkg="sensors" type="los.py" />

    <!--Launch rosbag to record topics -->
    <node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/alex/Documents/rasp_ws/src/sensors/bags/rl_g/los/w_speed/zz_1" />

</launch>
